app/utils/translations.js

```diff
@@ -0,0 +1,1011 @@
+const translations = {
```

Comment:
Probably out of scope for this PR, but it would be good to be thinking about `ember-i18n` integration in the near future. We can always change this data structure later, just something to keep in mind.